Automattic/wp-calypso
# Gutenberg: remove metaboxes related code from edit-post

Part of https://github.com/Automattic/wp-calypso/issues/26642

Starting the removal of unneeded pieces from Calypso's `edit-post` folder that were originally copied from Gutenberg core.

### Testing instructions
1. Navigate to `/gutenberg/post/{siteSlug}` and verify that the editor shell is still working as before.
2. Calypso smoke test.
